test/lit.cfg

```diff
@@ -53,7 +53,9 @@ def darwin_sdk_build_version_split(version):
 
 
 def darwin_sdk_build_version_cmp(lhs, rhs):
-    return cmp(
+    def _cmp(a, b):
```

Comment:
Oh yeah, totally. I saw the commit message. Sorry if it wasn't clear and seemed like I was blaming you. I was not.